pkg/authorization/registry/localresourceaccessreview/rest_test.go

```diff
@@ -37,12 +39,11 @@ func (a *testAuthorizer) Authorize(attributes kauthorizer.Attributes) (allowed b
 
 	return false, "", errors.New("Unsupported")
 }
-func (a *testAuthorizer) GetAllowedSubjects(passedAttributes kauthorizer.Attributes) (sets.String, sets.String, error) {
+
+func (a *testAuthorizer) AllowedSubjects(passedAttributes kauthorizer.Attributes) ([]rbac.Subject, error) {
```

Comment:
this one probably needs to be rbacauthorizer.Attributes (import k8s.io/kubernetes/plugin/pkg/auth/authorizer/rbac) and not kauthorizer.Attributes

Reply:
@simo5 that's odd, it looks like you got the errors from old changes. Maybe? It happened before because `AllowedSubjects` as not present inside `rest_test.go`. But that's not the case now.

Are there any chances that you're looking at the old change? For me `rest_test.go` is passing here. How did you get these errors? 

Also, `kauthorizer.Attributes` comes from `"k8s.io/apiserver/pkg/authorization/authorizer"`, which seems correct. Looking at `k8s.io/kubernetes/plugin/pkg/auth/authorizer/rbac`, there's no such function, perhaps I'm looking at the wrong place?

Reply:
@simo5 ignore my previous comments. Found it at the logs right now.

Reply:
@simo5 so after dig into it to understand what is really going on. Here's the confusing part about this failure:

1. Tests pass in Gogland, even after run a `make clean build`. 
2. Tests also pass from the command line
```
make test-unit WHAT=pkg/authorization/registry/localresourceaccessreview
```

In theory, the second step was supposed to fail.

Regards your comment about `rbacauthorizer.Attributes`, as far as I know, there's no such thing in to the API.

Reply:
ignore my comment about rbacauthorizer I mixed up files when looking at them